swagger.yaml

```diff
@@ -4214,6 +4214,11 @@ definitions:
         format: date-time
         x-go-custom-tag: gorm:"type:timestamp with time zone;default:'2000-01-01 00:00:00z'"
         description: The time that this cluster completed installation.
+      estimated_deletion_time:
```

Comment:
maybe we should call it estimated_deregistration_time because we will deregister it and after X days we will delete it from our DB

Reply:
I think this is implementation details, the user won't be able to access an unregistered cluster - from his point of view it doesn't exist anymore.

I don't mind changing it though.